src/main/java/lotto/Application.java

```diff
@@ -9,7 +9,8 @@ public static void main(String[] args) {
         ResultView.printLottos(lottos);
 
         WinningNumbers winningNumbers = InputView.askWinningNumbers();
-        LottoResults results = lottos.checkResults(winningNumbers);
+        int bonusNumber = InputView.askBonusNumber();
```

Comment:
지역 변수가 `lottos.checkResults()` 의 파라메터로 전달되는 것 이외에는 사용처가 없으므로, 
변수를 별도로 할당하지 말고 파라메터 안에서 바로 입력받는 구현도 괜찮을 것 같아요. 

```suggestion
        LottoResults results = lottos.checkResults(InputView.askWinningNumbers(), InputView.askBonusNumber());